db/migrate/20161119134300_add_organizer_to_workshops.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrganizerToWorkshops < ActiveRecord::Migration
+  def change
+    add_column :workshops, :organizer, :string
```

Comment:
The problem is, migrations that were merged earlier are already performed, so you cannot just squeeze yours in. Please rename the file and change the migration. Have a look at [this](http://edgeguides.rubyonrails.org/active_record_migrations.html) or go to the slack channel "ruby-help".